Dockerfile

```diff
@@ -20,7 +31,8 @@ COPY internal internal
 
 # Prepare static files to be embedded in Go binary
 RUN rm -rf internal/server/public_html
-COPY public_html internal/server/public_html
+COPY --from=builder-frontend /node/src/app/build internal/server/public_html
```

Comment:
Information from you about this is going good.

Reply:
N What are you doing for your family for the holidays? P You can save money on your account balance and your family if you want to be in a better connection.